### DISCUSSION TITLE
HS_STATUS if disconnecting

### DISCUSSION THREAD

**Discussion by Sill1000** [HomeSpan/HomeSpan#567] [Q&A] [answered]
Hello, 

I would like to turn on a led if the Wifi connection is lost. It seems that statusString(HS_STATUS s) does not have this information.
How to do that ?

Thanks

**Comment by HomeSpan** [comment, marked as answer]
Use HS_WIFI_CONNECTING.  This status occurs whenever HomeSpan is trying to connect to WiFi.  If WiFi is lost, HomeSpan automatically tries to reconnect, so you can use this to trigger an LED.

**Reply by Sill1000** [reply]
Perfect thanks !
